domains/maurom.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "MM-coder",
+           "email": "mauromartins@email.com",
+           "discord": "804668335039053874"
+        },
+    
+        "record": {
+            "A": ["185.199.108.153"]
```

Comment:
Unable to view site, are you able to provide a preview?

Reply:
Hello, that's the github pages IP, current site is https://maurom.dev